src/exception/exception-service.ts

```diff
@@ -185,4 +185,31 @@ export namespace operations {
       });
     }
   };
+
+  const deleteEntity = async (programId: string, entity: Entity) => {
+    let result, errorMessage;
+    if (entity) {
+      result = await entityExceptionRepository.deleteSingleEntity(programId, entity);
+      errorMessage = `no ${entity} entity exceptions for program '${programId}'`;
+    } else {
+      result = await entityExceptionRepository.delete(programId);
+      errorMessage = `no entity exceptions for program '${programId}'`;
+    }
+    return { result, errorMessage };
+  };
+
+  export const deleteEntityException = async ({
+    programId,
+    entity,
+  }: {
+    programId: string;
+    entity: Entity;
+  }) => {
+    const { result, errorMessage } = await deleteEntity(programId, entity);
+    return processResult({
+      // @ts-expect-error v3.9.5, no yelling in v4
```

Comment:
This line is marked as an error for me, not clear what comment means here

Reply:
does it build?
the comment just means that ts is expecting an error in ts version 3.9.5, working fine in ts v4

using this decorator for now as it's blocking getting code complete / pushed through review into testing as a priority

Reply:
Great yes it does build